src/GamePlayer.cpp

```diff
@@ -826,45 +826,50 @@ void GamePlayer::FindWarehouseForAllJobs(const Job job)
 
 Ware* GamePlayer::OrderWare(const GoodType ware, noBaseBuilding* goal)
 {
-    /// Gibt es ein Lagerhaus mit dieser Ware?
-    nobBaseWarehouse* wh = FindWarehouse(*goal, FW::HasMinWares(ware, 1), false, true);
-
-    if(wh)
+    if(emergency)
     {
-        // Prüfe ob Notfallprogramm aktiv
-        if(!emergency)
-            return wh->OrderWare(ware, goal);
-        else
-        {
-            // Wenn Notfallprogramm aktiv nur an Holzfäller und Sägewerke Bretter/Steine liefern
-            if((ware != GD_BOARDS && ware != GD_STONES) || goal->GetBuildingType() == BLD_WOODCUTTER
-               || goal->GetBuildingType() == BLD_SAWMILL)
-                return wh->OrderWare(ware, goal);
-            else
-                return NULL;
-        }
-    } else // no warehouse can deliver the ware -> check all our wares for lost wares that might match the order
+        if(ware == GD_STONES && goal->GetBuildingType() != BLD_SAWMILL)
```

Comment:
While this is correct for now, don't assume that woodcutters need no stones. I'm working on some code that makes the costs changeable. So just leave this as-is: Stones and boards ordered? Then return NULL for everything but woodcutter and sawmill.